src/ChatWindow.js

```diff
@@ -164,7 +163,11 @@ class ChatWindow extends Component {
     this.pushMessage(chatterIds.USER, userRequestToAlexa);
     this.setState({ userRequestToAlexa: "" });
 
-    const access_token = this.props.authenticationInfo.getAccessToken();
+    let access_token;
```

Comment:
This was probably a bad decision to let access_token to flow through. You don't need to fix because you didn't write but put a TODO to change the code to kick user back to logout screen if access_token is missing or invalid.

Reply:
Added TODO.